.github/workflows/codeql.yml

```
@@ -23,7 +23,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: [ 'javascript-typescript', 'php' ]
+        language: [ 'javascript-typescript' ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)